corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowser.java

```diff
@@ -0,0 +1,110 @@
+package org.corfudb.browser;
+
+import com.google.common.reflect.TypeToken;
+
+import org.slf4j.LoggerFactory;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.core.joran.spi.JoranException;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.CorfuDynamicKey;
+import org.corfudb.runtime.collections.CorfuDynamicRecord;
+import org.corfudb.runtime.view.TableRegistry;
+import org.corfudb.util.serializer.DynamicProtobufSerializer;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+
+/**
+ * This is the UFO Browser Tool which prints data in a given namespace and table.
+ *
+ * - Created by pmajmudar on 10/16/2019.
+ */
+@Slf4j
+public class CorfuBrowser {
+    private final CorfuRuntime runtime;
+    private final String singleNodeEndpoint;
+    private static final String LOG_LEVEL = "INFO";
+
+    /**
+     * Creates a CorfuBrowser which connects a runtime to the server.
+     * @param host Corfu Server Host
+     * @param port Corfu Server Port
+     * @param keyStore Location of the KeyStore File
+     * @param ksPasswordFile Location of the KeyStore Password File
+     * @param trustStore Location of the Truststore File
+     * @param tsPasswordFile Location of the Truststore Password File
+     * @param tlsEnabled Tls Enabled Status
+     */
+    public CorfuBrowser(String host, int port, String keyStore,
```

Comment:
An endpoint is actually a cluster, not a single address. I suggest you just pass a comma separated connection string. For example, ip1:port1,ip2:port2

Reply:
I based this on the current implementation of the browser tool in the NSX repo.  Will address this in a subsequent patch.